public/apps/account/account-nav-button.tsx

```diff
@@ -173,15 +174,12 @@ export function AccountNavButton(props: {
 
   // ToDo: Add aria-label and tooltip when isPlacedInLeftNav is true
   const innerElement = isPlacedInLeftNav ? (
-    <EuiButtonEmpty
-      size="xs"
-      flush="both"
-      className="accountNavButton"
```

Comment:
Since we are removing this `className` in the the new component, should we also remove its scss file - `public/apps/account/account-nav-button.scss`? Or do we need to apply the same styling to this new component?